src/Workspaces/CSharp/Portable/Formatting/Rules/BaseFormattingRule.cs

```diff
@@ -226,14 +227,32 @@ protected void AddBraceSuppressOperations(List<SuppressOperation> list, SyntaxNo
 
                 if (SomeParentHasMissingCloseBrace(node.Parent))
                 {
-                    endBrace = endBrace.GetPreviousToken();
+                    if (node.IsKind(SyntaxKind.Block) && ((BlockSyntax)node).Statements.Count >= 1)
```

Comment:
or perhaps do this cast before the if since you use it inside the if block

---
In reply to: [58458141](https://github.com/dotnet/roslyn/pull/10191#discussion_r58458141) [](ancestors = 58458141)